taskmanager/features/task.feature

```diff
@@ -26,3 +26,13 @@ Feature: Task management
     Given there is task named "Add switch language button" assigned to user named "Carl"
     When I assigned task named "Add switch language button" to user named "Sarah"
     Then task named "Add switch language button" should be assigned to user named "Sarah"
+
```

Comment:
can we add (can be as separate PR) feature related to default priority? That if i do not choose priority `Major` will be set?